src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -122,6 +122,7 @@ public ConnectionManagerSettings(NodeSettings nodeSettings)
             this.BurstModeTargetConnections = config.GetOrDefault("burstModeTargetConnections", 1, this.logger);
             this.SyncTimeEnabled = config.GetOrDefault<bool>("synctime", true, this.logger);
             this.RelayTxes = !config.GetOrDefault("blocksonly", DefaultBlocksOnly, this.logger);
+            this.IpRangeFiltering = config.GetOrDefault<bool?>("IpRangeFiltering", null, this.logger);
```

Comment:
You could default to `true` and make the IpRangeFiltering a `bool` rather than a `bool?` .

Also, can you please add the cmd line arg to `BuildDefaultConfigurationFile` and `PrintHelp`?

Reply:
You could default to true and make the IpRangeFiltering a bool rather than a bool?

bool? exists as default value is ambiguous
This is because IP range filtering has two default behaviours:
If peer is local host: false unless set explicitly to true.
if peer is remote host: true unless set to false.

Reply:
I agree with Jeremy, this should be true by default. Local networks should never be considered

Reply:
Done.